### PR TITLE
Preparing for GCS Connector 2.2.8 release

### PR DESCRIPTION
**Pull request by arunkumarchacko**

### CODE REVIEW

File: gcs/CHANGES.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
-### 2.2.8 - 2022-XX-XX
+### 2.2.8 - 2022-09-12
```
Comment (Deependra-Patel) [inline gcs/CHANGES.md:1]: Let's put all the changes 

like 
https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/802
https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/858

Comment (arunkumarchacko) [reply gcs/CHANGES.md:1]: Done

**Review by Deependra-Patel — APPROVED** [review]

**Pull request merged**